pages/en/blog/announcements/v22-release-announce.md

```diff
@@ -0,0 +1,97 @@
+which contains the full list of commits included in this release.
+
+### Notable Changes
+
+#### glob and globSync
+
+This release introduces `glob` and `globSync` functions for pattern matching.
+Developers can now utilize these functions for matching file paths based on specified patterns.
+
+Contributed by Moshe Atlow [#51912](https://github.com/nodejs/node/pull/51912)
+
+#### Maglev
+
+V8's Maglev Compiler is now enabled by default on supported architectures (<https://v8.dev/blog/maglev>).
+Maglev improves CPU performance for short-lived CLI programs.
+Contributed by Keyhan Vakil in [#51360](https://github.com/nodejs/node/pull/51360)
+
+#### Support require()ing synchronous ESM graphs
+
+This release adds `require()` support for synchronous ESM graphs under
+the flag `--experimental-require-module`.
+
+If `--experimental-require-module` is enabled, and the ECMAScript
+module being loaded by `require()` meets the following requirements:
+
+- Explicitly marked as an ES module with a "type": "module" field in the closest package.json or a .mjs extension.
+- Fully synchronous (contains no top-level await).
+
+`require()` will load the requested module as an ES Module, and return
+the module name space object. In this case it is similar to dynamic
+`import()` but is run synchronously and returns the name space object
+directly.
+
+Contributed by Joyee Cheung in [#51977](https://github.com/nodejs/node/pull/51977)
+
+#### Running package.json scripts
+
+Node.js 22 includes a new experimental feature for the execution of scripts from the `package.json` with the cli flag `node --run <script-in-package-json>`.
+It also supports `node run test` which executes the test command inside `package.json` scripts.
+Contributed by Yagiz Nizipli [#52190](https://github.com/nodejs/node/pull/52190)
+
+#### Stream default High Water Mark
+
+Increased the default High Water Mark for streams from 16KiB to 64KiB.
+This provides a performance boost across the board at the cost of slightly higher memory usage.
+Users in memory sensitive environments are encouraged to explicitly set `setDefaultHighWaterMark`.
+Contributed by Robert Nagy in [#52037](https://github.com/nodejs/node/pull/52037)
+
+#### Watch Mode (node --watch)
+
+From this release Watch Mode is considered stable.
+When in watch mode, changes in the watched files cause the Node.js process to restart.
+
+Contributed by Moshe Atlow [#52074](https://github.com/nodejs/node/pull/52074)
+
+#### Websocket
+
+The browser-compatible implementation of WebSocket, previously behind the flag `--experimental-websocket`, will be enabled by default.
+This provides a WebSocket client to Node.js without external dependencies.
+
+Contributed by the Undici team and Aras Abbasi in [#51594](https://github.com/nodejs/node/pull/51594)
+
+### Improve performance of AbortSignal creation
+
+This release improves the performance of `AbortSignal` instances creation by 75% for regular use and 2000% for other cases.
```

Comment:
Should we provide a link to benchmark? @rluvaton I tried to come up with a notable change description

Reply:
I would just remove the numbers. Mention there's a performance improvement on AbortSignal

Reply:
I don't think we should document the 2000% as it's only in very certain case that I don't think anyone will use

Reply:
And also, can you use the updated version?

```
Creating AbortSignal’s have been significantly faster. This change results in improving fetch performance as well as the test runner performance.
```

Reply:
LGTM